src/cli/lint.ts

```
@@ -0,0 +1,97 @@
+import type { Options } from '@cspell/eslint-plugin'
+import cspellRecommended from '@cspell/eslint-plugin/recommended'
+import js from '@eslint/js'
+import react from '@eslint-react/eslint-plugin'
+import { Command } from 'commander'
```

ESLint

[error] Unable to resolve path to module 'commander'.

(import-x/no-unresolved)